### ISSUE TITLE
npm is still v0.1 without client/server

### ISSUE THREAD

**Issue by Head** [aprescott/tenuki.js#15]
Can you please make a npm release for the new client/server stuff?

**Comment by aprescott** [comment]
@Head I'm planning on cutting a new release within the next day, to give me enough time to maybe include a few other in-progress features. If you're itching to try it out right away, you can temporarily work against the git repo.

If you have any feedback on the `Client` interface, please feel free to open a GitHub issue about anything you come across that's awkward, or broken, or confusing. I'm hoping to eventually provide enough functionality that it's possible to use `Client` to write a full client against something like online-go.com. I unfortunately couldn't find a JavaScript client for their API, so I've only tested it with my dummy server. It'd be great to have more real-world info.

**Comment by Head** [comment]
I found out how to npm install the master branch, but it lacks the /src/ files. Then I tried bower and found out: You renamed them from /lib/ to /src/ and let the index.js file with ./lib/ - looks like this make npm to fail. I'll try further stuff tomorrow. Thx for the quick reply!

**Comment by aprescott** [comment]
Whoops, that's a glaring oversight on my part. I'll make sure it gets fixed!

**Comment by Head** [comment]
Ok, I need to surrender. I can't get the latest version running, whatever I do.
If I npm install aprescott/tenuki.js there are the error with index.js and in .npmignore is the /src/ folder.
I cloned your repo and did "npm install Head/tenuki.js". Then I fire up my MEAN.IO project with gulp and get a "game.js:1 Uncaught SyntaxError: Unexpected token import".

Ok, let's try to just download and use your build.js with this webkit thing:
Now I get "Cannot find module "tenuki"" in my angular controller when I run "var tenuki = require("tenuki");" and "tenuki is not defined" without.

**Comment by aprescott** [comment]
@Head for now, when you work off the repo, install the dependencies and run `./build.sh` and it'll compile `lib/`. After that, you should be able to `var tenuki = require("./index.js");`. At the very least, I need to improve the setup instructions for working off a cloned repo.

**Comment by aprescott** [comment]
```shell
npm install
./build.sh
node
```

```js
tenuki = require("./index");
tenuki.Game;
// [Function: Game]
```

**Comment by aprescott** [comment]
@Head I've published v0.2.0 with `Client` included (among [other changes](https://github.com/aprescott/tenuki.js/blob/master/CHANGELOG.md#v020)), so I'm going to close this. Installing from the git repo with `npm install aprescott/tenuki.js` is still broken while I decide on a good way of solving it.

Again, feel free to open any issues you find!
